aws_quickstart/datadog_agentless_scanning.yaml

```diff
@@ -570,6 +570,20 @@ Resources:
             Condition:
               StringNotEquals:
                 'aws:ResourceTag/DatadogAgentlessScanner': 'false'
+          - Sid: DatadogAgentlessScannerECRAuthorizationToken
+            Action:
+              - "ecr:GetAuthorizationToken"
```

Comment:
Let's add the permissions for ECR public images:
`ecr-public:GetAuthorizationToken`

I didn't test it but we also probably need `sts:GetServiceBearerToken` too for public ECR (according to https://docs.aws.amazon.com/AmazonECR/latest/public/public-repository-policies.html)